rdftk_io/src/xml/reader.rs

```diff
@@ -482,6 +485,7 @@ fn parse_object_element<R: Read>(
                 if has_elements {
                     error_event!(state => "parse_object_element", &format!("found character content after element(s)"));
                 }
+                #[allow(unused_assignments)]
                 has_characters = true;
```

Comment:
Should these simply be removed?

Reply:
If there's no (future?) purpose for `has_characters` then yes I'd remove it